src/MarkerClusterGroup.js

```diff
@@ -260,6 +260,9 @@ L.MarkerClusterGroup = L.FeatureGroup.extend({
 				m = layersArray[i];
 				this._arraySplice(this._needsClustering, m);
 				npg.removeLayer(m);
+				if (this.hasLayer(m)) {
```

Comment:
To be honest I'm not sure if we really need that check. The important part is that the layer is added to the _needsRemoving array. 
The check is there when removing a single Layer so I added it here too.